portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,47 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container"> 
+               
+            <div id="intro">
+        <h1>Erik Rodriguez</h1>
+            <p>
+                Hi everyone my name is Erik, 
+                I am a sophomore Software Engineering Student at The University of Texas at Dallas 
+                I am passionate about technology, leadership and designing products.  
+                My goal is to become a Software Engineer and have meaningful impact solving problems around the world.
+            </p>
+
```

Comment:
nitpick: only one space needed here for readability. Same for rest of file as well.